compiler/ras/LimitFile.cpp

```diff
@@ -687,14 +693,18 @@ TR_Debug::limitfileOption(char *option, void *base, TR::OptionTable *entry, TR::
          }
       if (limitFileError)
          {
+         TR_VerboseLog::vlogAcquire();
          TR_VerboseLog::writeLine(TR_Vlog_FAILURE, "Bad limit file entry --> '%s'", limitReadBuffer);
+         TR_VerboseLog::vlogRelease();
          return fail;
          }
       fclose(limitFile);
       }
    else
       {
+      TR_VerboseLog::vlogAcquire();
       TR_VerboseLog::writeLine(TR_Vlog_FAILURE, "Unable to read limit file --> '%s'", limitFileName);
+      TR_VerboseLog::vlogRelease();
```

Comment:
Can these be `writeLineLocked` calls instead? Similarly elsewhere we have the pattern:

```
TR_VerboseLog::vlogAcquire();
TR_VerboseLog::writeLine(...);
TR_VerboseLog::vlogRelease();
```